test/tests/performance/pbs_runjobwait_perf.py

```diff
@@ -125,3 +128,9 @@ def test_rw_runjobhook_nohook(self):
         # the time taken by none mode, as without a runjob hook, the
         # scheduler should assume none mode even if job_run_wait=runjob_hook
         self.assertLess(t_rj / t_none, 1.5)
+        self.perf_test_result(t_rj, "time_taken_rw_runjobhook_nohook", "sec")
```

Comment:
time taken only by runjob hook

Reply:
Time taken by runjob hook is a different case. It is a case where actually no hook present.